sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -184,7 +174,7 @@ To try different pipeline topologies with the SDK, please see the official [Samp
 
 ## Troubleshooting
 
-- When sending a method request using the IoT Hub's `CloudToDeviceMethod` remember to not type in the method request name directly. Instead use `[MethodRequestName.method_name]`
+- When sending a method request using the IoT Hub's `CloudToDeviceMethod` remember to not type in the method request name directly. Instead use `[MethodRequestName.methodName]`
```

Comment:
Properties in .NET / C# - all public members, actually - should be PascalCase. Also, this isn't a valid link. If you just want an inline codeblock, use only backticks, e.g.

```
`MethodRequestName.MethodName`
```